### ISSUE TITLE
Does REViewer plot reads from off-target regions?

### ISSUE THREAD

**Issue by bw2** [Illumina/REViewer#3]
I ran EHv4 on the FXN locus, using 2 repeat specs which are identical except that the 2nd one includes GAA off-target regions:
```
{
        "LocusId": "FXN-chr9-69037286-69037304-GAA",
        "LocusStructure": "(GAA)*",
	"RepeatUnit": "GAA",
        "ReferenceRegion": "chr9:69037286-69037304",
        "VariantType": "RareRepeat",
	"OfftargetRegions": []
    },
    {
        "LocusId": "FXN-chr9-69037286-69037304-GAA-with-off-targets",
        "LocusStructure": "(GAA)*",
	"RepeatUnit": "GAA",
        "ReferenceRegion": "chr9:69037286-69037304",
        "VariantType": "RareRepeat",
	"OfftargetRegions": [
	    "chr2:220546033-220546610",
	    "chr5:127247161-127247640",
	    "chrX:51621350-51621856",
	    "chr1:101657701-101658187",
	    "chr13:102161416-102161881",
	    "chr7:37848005-37848522",
	    "chrY:25645531-25646013",
	    "chr7:84690949-84691442",
	    "chrUn_KN707747v1_decoy:1062-2074",
	    "chr6:50708070-50708556",
	    "chrY:24024122-24024600"
	]
    },
```

These are the EHv4 results in the (relatively rare) WGS sample where the genotypes from the 2 specs differed significantly between no-off-targets:
```
chr9	69037286	.	A	<STR9>,<STR110>	.	PASS	END=69037304;REF=6;RL=18;RU=GAA;VARID=FXN-chr9-69037286-69037304-GAA;REPID=FXN-chr9-69037286-69037304-GAA	GT:SO:REPCN:REPCI:ADSP:ADFL:ADIR:LC	1/2:SPANNING/INREPEAT:9/110:9-10/63-153:2/0:5/13:0/12:42.016851
```
and with off-targets:
```
chr9	69037286	.	A	<STR33>,<STR726>	.	PASS	END=69037304;REF=6;RL=18;RU=GAA;VARID=FXN-chr9-69037286-69037304-GAA-with-0.01-threshold-off-targets;REPID=FXN-chr9-69037286-69037304-GAA-with-0.01-threshold-off-targets	GT:SO:REPCN:REPCI:ADSP:ADFL:ADIR:LC	1/2:FLANKING/INREPEAT:33/726:33-111/658-1313:0/0:13/13:4/108:42.016851
```

I then ran REViewer for both outputs, and got these plots

no-off-targets:
![CDS-nC6iXU_FXN-chr9-69037286-69037304-GAA_ExpansionHunter4](https://user-images.githubusercontent.com/6240170/104143695-92e6e900-538e-11eb-9cbe-2dfcad43ffde.png)

with-off-targets:
![CDS-nC6iXU_FXN-chr9-69037286-69037304-GAA-with-0 01-threshold-off-targets_ExpansionHunter4](https://user-images.githubusercontent.com/6240170/104143703-9da17e00-538e-11eb-97f4-a9b5dc8e61f5.png)

finally, this is the plot from when I used the standard FXN repeat spec included in the EHv4 repo:
![CDS-nC6iXU_FXN-chr9-69037286-69037304-GAA-official_ExpansionHunter4](https://user-images.githubusercontent.com/6240170/104143746-c0339700-538e-11eb-886b-b0d0dd08129e.png)

I'm wondering how to interpret the "with-off-targets plot".  In REViewer docs, I saw
`..the current version of REViewer visualizes repeats whose span does not exceed the fragment length (longer repeats are capped at the fragment length).`
Does REViewer not plot the off-target FRRs?
Also, would it be possible and/or useful to plot read-pairs rather than reads?

Thanks
-Ben

**Comment by egor-dolzhenko** [comment]
That's right. The current version of EH does not generate graph alignments of in-repeat read pairs (both mates inside the repeat). To account for the absence of these reads, REViewer caps the repeat size estimate at the fragment length. However, your example (with-off-targets) shows that sometimes the size of the repeat needs to be reduced further.

For now, we could cap the repeat size in a better way that would make both plots (with-off-targets and no-off-targets) look the same. Of course, a better solution would be to enable EH to output alignments of in-repeat read pairs.

**Comment by bw2** [comment]
I see, that makes sense. Thanks.
